rocAL/include/api/rocal_api_types.h

```diff
@@ -376,4 +376,10 @@ enum RocalExternalSourceMode {
     ROCAL_EXTSOURCE_RAW_UNCOMPRESSED = 2,
 };
 
+enum RocalAudioBorderType {
```

Comment:
Add brief description and Doxygen comments, refer other enums

Reply:
Address this.

Reply:
Still not added, Please check

Reply:
added brief desc

Reply:
@swetha097 You have added only the enum names in the comments. Please add a description about the border Type.

Reply:
Please address this.